db/seeds.rb

```diff
@@ -5,3 +5,32 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Passenger.delete_all
+Booking.delete_all
+
+Airport.delete_all   
+lag = Airport.create(city: "LAG", name: "Muritala Mohammed Airport Lagos")
+abj = Airport.create(city: "ABJ", name: "Nnamdi Azikiwe Airport Abuja")
+phc = Airport.create(city: "PHC", name: "Port Harcourt International Airport")
+kad = Airport.create(city: "KAD", name: "Kaduna International Airport")
+enu= Airport.create(city: "ENU", name: "Enugu International Airport")
```

Comment:
Useless assignment to variable - `enu`.<br>Surrounding space missing for operator `=`.